### ISSUE TITLE
Got an error following installation+example guides

### ISSUE THREAD

**Issue by tinchodias** [alvarop100/Storm2#10]
Hi, I just tried to explore the project the first time. First, I downloaded Pharo 7 64bits, and then evaluated:

~~~
Metacello new
  baseline: 'Winter';
  repository: 'github://ba-st/Winter:release-candidate/source';
  load.
~~~

and then:

~~~
ArkanoidGame new start
~~~

which couldn't find an asset. I see that in there are assets for the example games in the file structure of this repository, but as I can wait, I prefer to report the issue here... maybe I just did something wrong.

Thanks.

**Comment by apiorno** [comment]
Hi,
here https://github.com/ba-st/Winter/blob/release-candidate/docs/Examples.md you can see instructions.
Inside **assets/Game Examples Files** you have assets for all example games.
You must copy assets from **assets/Game Examples Files/arkanoid** to image directory ( i think only assets inside but not the entire folder).
You will face some performance issues with Arkanoid and also with Pharo 7(Working on that now).
Let me know if you need more help

**Comment by tinchodias** [comment]
Thanks! 

I guess I didn't read the sentence about copying the assets to the image directory. 
However, after opening the issue, I just modified some code in my image to read the assets directly from the iceberg directory where the repository is cloned. Not sure if it's a good solution, but I can copy the key method here for the case it has some interest:

~~~
"accessing"
WinterAsset class >>
basePath
	^ basePath ifNil: [ (FileLocator localDirectory / #iceberg / 'ba-st' / 'Winter' / 'assets') asFileReference ]
~~~

Then, I appended this `WinterAsset basePath` in a couple of Winter methods in order to generate the expected directory where the assets are.

So, we can close this issue, no?
